### PR TITLE
Add admin job api, it supports Harbor admin to trigger job

### PR DESCRIPTION
**Pull request by wy65701436**
either manual or schedule. The job will be populated to job
service to execute. The api includes 1. POST /api/admin/jobs
2, GET /api/admin/jobs/?name 3, GET /api/admin/jobs/:id/log.

### CODE REVIEW

File: src/common/dao/admin_job.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+// GetAdminJob ...
+func GetAdminJob(id int64) (*models.AdminJob, error) {
+	o := GetOrmer()
+	aj := models.AdminJob{ID: id}
+	err := o.Read(&aj)
+	if err == orm.ErrNoRows {
+		return nil, nil
+	}
```
Comment (ywk253100) [inline src/common/dao/admin_job.go]: else?

File: src/ui/api/admin_job.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+		return
+	}
+
+}
+
+//Post ...
+func (aj *AdminJobAPI) Post() {
+
+	ajr := models.AdminJobReq{}
+	aj.DecodeJSONReq(&ajr)
```
Comment (ywk253100) [inline src/ui/api/admin_job.go]: Call DecodeJSONReqAndValidate(), it will decode and validate the request data.

File: src/ui/api/admin_job.go [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+}
+
+// submitJob submits a job to job service per request
+func (aj *AdminJobAPI) submitJob(ajr models.AdminJobReq) {
+	id, err := dao.AddAdminJob(&common_models.AdminJob{
+		Name: ajr.Name,
+		Kind: ajr.Kind,
+	})
+	if err != nil {
+		aj.HandleInternalServerError(fmt.Sprintf("%v", err))
```
Comment (ywk253100) [inline src/ui/api/admin_job.go]: Should we update the job status in the database if got error?

Comment (ywk253100) [reply src/ui/api/admin_job.go]: Should consider the scenario that users submit the GC job more than once, and how can we determine the status of the GC process?

**Comment by wy65701436** [comment]
Close it as API change, and use https://github.com/vmware/harbor/pull/5344 to track.

**Pull request closed without merging**